### PR TITLE
Changed outdated type-function to is_<type>.

### PR DESCRIPTION
**Pull request by ejabach**

### CODE REVIEW

**Comment by ejabach** [comment]
Forgot 2 of the  3 files. I'll update the Pull Request as soon as I fixed all files.

File: manifests/idmap.pp [hunk 1]
```diff
@@ -35,50 +35,32 @@
...
       $translation_method_real = inline_template('<%= translation_method.join(\',\') %>')
       # GH: TODO: write valid regex
-    }
-    default: {
+  } else {
       fail("valid types for translation_method are String and Array. Detected type is <${translation_method_type}>")
-    }
   }
-
+
```
Comment (ghoneycutt) [inline manifests/idmap.pp]: Please remove the trailing whitespace

**Comment by ghoneycutt** [comment]
Could you please rebase your commits into one commit.

**Comment by ejabach** [comment]
I rebased the commits and deleted the trailing whitespace within my last commit. In github it still says that the whitespace exists, but as soon as I fetch this commit it is gone. And the checks didn't start from new as well.
I'm new to git so it could be that I failed in the rebase process...

**Comment by Phil-Friderici** [comment]
Thanks @ejabach, looks good to me.
Hope @ghoneycutt is fine with an additional commit for fixing ruby 1.8.7 here.

**Comment by Phil-Friderici** [comment]
@ejabach: in the meantime, Puppet v4 compatibilty was also reached with this commit: https://github.com/ghoneycutt/puppet-module-nfs/commit/693f16edb2b6391d417833d5bc602e6182c9ea31

Therefore, I would recommend to close this PR.

**Pull request closed without merging**